projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -149,3 +149,6 @@ datajobs.security.kerberos.kerberosPrincipal=""
 datajobs.security.kerberos.keytabFileLocation=
 datajobs.security.kerberos.krb5ConfigLocation=
 datajobs.security.kerberos.enabled=false
+
```

Comment:
```#Settings for Kerberos authorization provider.
datajobs.security.kerberos.kerberosPrincipal=""
```
Principal, and the rest of the properties you can trace, are actually used to construct KerberosServiceAuthenticationProvider (SunJaasKerberosTicketValidator);
Those properties have to do with authentication rather authorization.

Authorization is performed via configurable authorization webhook, for example find configurables:
```
featureflag.authorization.enabled=true
datajobs.authorization.webhook.endpoint=https://httpbin.org/post
```